avm/res/network/public-ip-address/main.bicep

```diff
@@ -159,7 +165,7 @@ resource publicIpAddress 'Microsoft.Network/publicIPAddresses@2023-09-01' = {
         }
       : null
     idleTimeoutInMinutes: idleTimeoutInMinutes
-    ipTags: null // Note: Requires feature 'Microsoft.Network/AllowBringYourOwnPublicIpAddress' to be registered on the subscription
```

Comment:
I take it the feature is now publically available and would not need to be registered in the subscription anymore?

Reply:
It looks like it's GA already a while (https://azure.microsoft.com/en-us/blog/bring-your-own-ip-addresses-byoip-to-azure-with-custom-ip-prefix/). I didn't have to register anything.